python/stressng/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM quay.io/centos/centos:stream8
+
+RUN dnf module -y install python39 && dnf install -y python39 python39-pip && dnf -y install stress-ng
```

Comment:
should we pin a specific version for stressng?

Reply:
@sanjacodes Thoughts?

Reply:
Uhm, did you mix up comments by chance?

Reply:
I think our standard needs to be that external tools are version-locked in our plugins. We've definitely seen problems in our other wrapper-type systems before where we haven't locked versions and feature/bug drift has broken our wrapper.

Reply:
done